app/models/megaphone/publisher.rb

```diff
@@ -24,7 +26,7 @@ def publish!
     def sync_episodes!
       Rails.logger.tagged("Megaphone::Publisher#sync_episodes!") do
         # delete or unpublish episodes we aren't including in the feed anymore
-        unpublish_and_delete_episodes!
+        delete_episodes!
```

Comment:
megaphone doesn't have an "archived" state - we could unpublish and leave a draft, but that's problematic as well, as those get used for inventory (esp for future episodes).